lld/COFF/Driver.cpp

```diff
@@ -1822,6 +1822,8 @@ void LinkerDriver::linkerMain(ArrayRef<const char *> argsArr) {
         ltoDebugPM = true;
       } else if (s == "noltodebugpassmanager") {
         ltoDebugPM = false;
+      } else if (s == "emitllvm") {
```

Comment:
I'd defer to @rnk or someone else about whether `/opt:emitllvm` is the most natural/idiomatic way of setting an option like this on the `link.exe` like interface? AFAIK the `/opt:` namespace is for configuring optimizations, right? We have a bunch of other lld-link specific options either named plain `/customoption` or `/lldcustomoption`.

Secondly, while working on this, would you consider adding `emit-asm` in some form as well?

Reply:
I have implemented emit-asm in https://github.com/llvm/llvm-project/pull/67079 .

Reply:
> I thought about this, but the best argument I have got so far is that this matches the equivalent option for the ELF linker, where it's in the `--plugin-opt=` umbrella.

Well, `--plugin-opt=` means "plugin options". But `/opt:` in (lld-)link means "optimization options", so I don't think it's the best fit. Or can @rnk chime in here if I'm misunderstanding things?